OpenXmlPowerTools/PtOpenXmlUtil.cs

```diff
@@ -678,13 +678,15 @@ public static int CalcWidthOfRunInTwips(XElement r)
             if (rPr == null)
                 throw new OpenXmlPowerToolsException("Internal Error, should have run properties");
             var languageType = (string)r.Attribute(PtOpenXml.LanguageType);
-            decimal? sz = null;
+            decimal? szn = null;
             if (languageType == "bidi")
-                sz = (decimal?)rPr.Elements(W.szCs).Attributes(W.val).FirstOrDefault();
+                szn = (decimal?)rPr.Elements(W.szCs).Attributes(W.val).FirstOrDefault();
```

Comment:
I don't think this will work. W.val is a double, so default will end up being `0`.

Reply:
This part of the code is the same as before, I only introduced a non nullable decimal after this.

Reply:
ah, you're right. It's a bug, but not a regression :)

Reply:
Can you create an issue to track this?